install_ffmpeg.sh

```diff
@@ -5,11 +5,22 @@ set -ex
 ROOT="${1:-$HOME}"
 ARCH="$(uname -m)"
 NPROC=${NPROC:-$(nproc)}
+EXTRA_CFLAGS=""
+EXTRA_LDFLAGS=""
+EXTRA_FFMPEG_FLAGS=""
 
 if [[ $ARCH == "arm64" ]] && [[ $(uname) == "Darwin" ]]; then
   # Detect Apple Silicon
   IS_M1=1
 fi
+if [[ $ARCH == "x86_64" ]] && [[ $(uname) == "Darwin" ]] && [[ "${GOARCH:-}" == "arm64" ]]; then
```

Comment:
is the `$ARCH` check needed here?

Reply:
I think the only consequence would be that this block would run during native M1 ARM64 builds, resulting in ffmpeg having `--enable-cross-compile` enabled even though it doesn't need it. I'm not 100% sure what that flag does.